tripal_chado/api/tripal_chado.mviews.api.inc

```diff
@@ -40,23 +40,36 @@
  *   admin pages. However, when called by Drush we don't want to redirect. This
  *   parameter allows this to be used as a true API function.
  *
+ * @return 
+ *   TRUE if the materialized view was successfully addd, FALSE otherwise.
  * @ingroup tripal_mviews_api
  */
 function chado_add_mview($name, $modulename, $mv_schema, $query, $comment = NULL, $redirect = TRUE) {
 
   if (!array_key_exists('table', $mv_schema)) {
      tripal_report_error('tripal_chado', TRIPAL_ERROR,
        'Must have a table name when creating an mview.', array());
-     return NULL;
+     return FALSE;
   }
 
   $mv_table = $mv_schema['table'];
-
+ 
   // See if the mv_table name already exsists.
-  $mview_id = db_query(
-    'SELECT mview_id FROM {tripal_mviews} WHERE name = :name',
-    array(':name' => $name))->fetchField();
-
+  $mview_id = db_select('tripal_mviews', 'tm')
+    ->fields('tm', array('mview_id'))
```

Comment:
old habits will be hard to break, but `['mview_id']`.

edit oh, i see.  youre sticking with the old style for this because you'd have to change the whole file otherwise.  I guess thats preferred, nevermind.

Reply:
I think we should consider the currently coding style of the project the old style until your PR goes through changing the full code base. In order words, checking that checkbox means you've kept the coding style consistent with the current state of the code ;-p